usr/sbin/p11sak/p11sak.c

```diff
@@ -9513,12 +9526,18 @@ static CK_RV init_pkcs11(const struct p11sak_cmd *command)
                   "'-N'/'--no-login'");
             return CKR_ARGUMENTS_BAD;
         }
+        if (opt_so) {
+            warnx("Option ' --so' is not allowed with '-N'/'--no-login'");
```

Comment:
Remove blank in ' --so' ?

Reply:
fixed (also a few lines above: `warnx("Option ' --force-pin-prompt' is not allowed with "` ).